t/24-worker-overall.t

```diff
@@ -492,7 +492,7 @@ subtest 'is_qemu_running' => sub {
     ok OpenQA::Worker::is_qemu('/usr/bin/qemu-system-x86_64'), 'QEMU executable considered to be QEMU';
     ok !OpenQA::Worker::is_qemu('/usr/bin/true'), 'other executable not considered to be QEMU';
 
-    my $pool_directory = tempdir('poolXXXX');
+    my $pool_directory = tempdir('/tmp/poolXXXX', CLEANUP => 1);
```

Comment:
I think the `CLEANUP` is not needed. The code calls `File::Temp->newdir()` which defaults to `CLEANUP => 1`